src/WidgetRegistry.ts

```diff
@@ -53,6 +53,13 @@ export function isWidgetBaseConstructor(item: any): item is WidgetBaseConstructo
 	return Boolean(item && item._type === WIDGET_BASE_TYPE);
 }
 
+function getEventType(label: RegistryLabel): RegistryLabel {
```

Comment:
Ah I think I get what you mean now, let me see what I can do to address that - it is the extra context that you can never provide with symbol events as it would always have to be assumed as `loaded`. So a string label could support `unloaded` (just an example) but we couldn't do that for symbols.